{{cookiecutter.project_slug}}/src/{{cookiecutter.project_module}}/settings.py

```diff
@@ -46,8 +46,9 @@ def __init__(self):
         """
         self.DEBUG = True
         self.SECRET_KEY = os.urandom(24)
+        self.SCRIPT_NAME = os.environ.get('SCRIPT_NAME', '/')
```

Comment:
`SCRIPT_NAME` is part of wsgi `environ`, it doesn't need to be set in the settings

Reply:
Did you test that? I thought I tried without and it wasn't working properly then but that day is kind of a blur to me.

Reply:
Yes I tested it :)